src/test/java/org/junitpioneer/testkit/ExecutionResults.java

```diff
@@ -55,6 +57,20 @@
 				.execute();
 	}
 
+	ExecutionResults(Class<?> testClass, String testMethodName, Class[] methodParameterTypes) {
```

Comment:
This constructor seems unnecessary - you could just join the canonical class names in the factory method and call the constructor that takes a `String`?

Reply:
> This constructor seems unnecessary - you could just join the canonical class names in the factory method and call the constructor that takes a `String`?

Its explicit requested by @nicolaiparlog . I could not find a place where I could use it, but as he requested it he most probably has something/somewhere in mind to use it.